.travis.yml

```diff
@@ -1,12 +1,20 @@
 language: cpp
 compiler: gcc
 cache: apt
-before_install:
-  - sudo add-apt-repository ppa:ubuntu-toolchain-r/test -y
-  - sudo add-apt-repository ppa:apokluda/boost1.53 -y
-  - sudo apt-get update -qq
-  - if [ "$CXX" = "g++" ]; then sudo apt-get install -qq g++-4.8; fi
+addons:
+  apt:
+    sources:
+      - boost-latest
+      - ubuntu-toolchain-r-test
+    packages:
+      - gcc-4.8
```

Comment:
Isn't g++-4.8 enough, if gcc is also needed g++ probably has it as an install dependency?